config/config.go

```diff
@@ -40,12 +40,13 @@ func LoadConfig(configFile string) Config {
 	return cfg
 }
 
-func (cfg Config) GetEnv(env string) EnvDirSection {
+func (cfg Config) GetEnv(env string) (EnvDirSection, error) {
 	if conf, ok := cfg.Sections[env]; ok {
 		conf.ParsePerms()
-		return *conf
+		return *conf, nil
```

Comment:
Thanks for the suggestion! I went ahead and made the change. Mid reviewing again?